.eslintrc.cjs

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	extends: ['./node_modules/@balena/lint/config/.eslintrc.js'],
+	parserOptions: {
+		project: 'tsconfig.js.json',
+		sourceType: 'module',
+	},
+	env: {
+		jest: true,
```

Comment:
We don't use jest?

Reply:
changed to `mocha: true` as accordingly to https://eslint.org/docs/latest/use/configure/language-options it sets the
> mocha - adds all of the Mocha testing global variables.` 

Without the setting the `it` `describe` etc. namespace is not known to eslint

Reply:
Why is this needed here but not in eg the api where we also use mocha and it lints fine?